packages/client/hmi-client/src/page/project/components/tera-project-overview.vue

```diff
@@ -228,7 +230,9 @@ const resources = useResourcesStore();
 const isEditingProject = ref(false);
 const inputElement = ref<HTMLInputElement | null>(null);
 const newProjectName = ref<string>('');
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const visible: Ref<boolean> = ref(false);
```

Comment:
Unused - causing the lint error

Reply:
No you're right... thats causing the lint error, and our linter is choosing to ignore the `disable` above.... hmmm

Reply:
`visible` _is_ being used though at the bottom of the file.